src/components/LandingHowItWorks/index.tsx

```diff
@@ -168,16 +168,40 @@ export const LandingHowItWorks: FC = () => (
         "py-8 sm:py-12 md:py-16",
       ].join(" ")}
     >
+      <aside
+        className={[
+          "p-8 mx-auto mb-16 md:mb-24 max-w-prose",
+          "border border-green shadow-green",
+        ].join(" ")}
+      >
+        <img
+          src='/images/illustrations/microcontroller-2.svg'
+          alt='Microcontroller'
+          className='inline-block mr-4 align-top'
+        />
+        <p className='mt-3'>
+          In Berlin gibt es ein offenes LoRaWan-Netzwerk, welches seit 2017
```

Comment:
I'm not really convinced by this element. It kinda floats in the middle, has no title, and seems added quickly afterwards (which it is). I really like its content though. I would try to integrate it more into the page, add a title. For example, you could try to make it look like a sidenote in a book "What is LoRaWAN?" or "LoRaWHAT??" and have a question mark in the illustration of the sensor, as if it was there for the extra curious minds visiting the page that what to know more.

Reply:
I agreed with @dnsos on a solution which we hope will also convince you --> he will add another commit after mine

Reply:
You're right @vogelino, this element deserved more attention. How about adding it next to the text where TTN is referenced many times?

<img width="1297" alt="Screenshot 2021-07-19 at 15 38 06" src="https://user-images.githubusercontent.com/15640196/126168780-3978e73f-d039-4321-9ac1-b49269dc8c4b.png">

I also added links for further reading.

Reply:
In my impression, the sensor with an added question mark would be too much because we already have a really "question-y" look through the headline now.

Reply:
Thx! Much better, great! I think that is perfect! (almost, if you allow me to "Meckern auf höchsten Niveau") :D

I like the "More coming soon" label but I think it could be better vertically aligned.

I also like that you placed the block right to the headline and title but its width makes it look off the grid, especially because the columns below aren't aligned with the block itself. What about making the green block and the dark column the same width? Or as an alternative left-align the green block to the dark column below?

Reply:
@vogelino, I liked your "Höchstes Niveau" suggestions about the label and the info box. See my latest commits for the implementation. What do you think?

Reply:
Great! Much better! Thx for adjusting! I feel it's like 😙👌 Mouahh